src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
use your class variable in order not to create new Random object each time method is called
```suggestion
        int index = random.nextInt(Color.values().length);
```